src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable/PeopleTable';
+import { Person } from '../types/Person';
+import { getPeople } from '../api';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [loading, setLoading] = useState(false);
```

Comment:
this component shows the table just for a moment at first render before set `isLoading` to `true`, try to set `true` value as default